dd-sdk-android-androidx-fragment/src/test/kotlin/com/datadog/android/androidx/fragment/StrategyTest.kt

```diff
@@ -22,7 +22,7 @@ import org.mockito.quality.Strictness
     ExtendWith(ForgeExtension::class)
 )
 @MockitoSettings(strictness = Strictness.LENIENT)
-internal abstract class LifecycleCallbacksTest {
+internal abstract class StrategyTest {
```

Comment:
Why do you call that "StrategyTest", when it's testing `XXXFragmentLifecycleCallbacks` classes ? 
You can rename that `AbstractFragmentLifecycleCallbacksTest`

Reply:
it is also the strategy test that inherits from this one. I will find a proper name.